great_expectations/rule_based_profiler/domain_builder/categorical_column_domain_builder.py

```diff
@@ -35,10 +47,12 @@ class CategoricalColumnDomainBuilder(ColumnDomainBuilder):
     This DomainBuilder uses column cardinality to identify domains.
```

Comment:
Can you move these to the if TYPE_CHECKING block?
```
from great_expectations.validator.computed_metric import MetricValue
from great_expectations.rule_based_profiler.parameter_container import ParameterContainer
```

Reply:
@Kilo59 Done -- thanks.